### ISSUE TITLE
Access is denied.

### ISSUE THREAD

**Issue by massob** [drizzle-mizzle/CharacterAI-Discord-Bot#81]
for some reason i start to get this 

**10:19:23 Discord     Discord.Net v3.10.0 (API v10)
10:19:23 Gateway     Connecting
10:19:29 Gateway     Connected
10:19:30 Gateway     Ready**
_System.ComponentModel.Win32Exception (5): Access is denied.
   at System.Diagnostics.NtProcessManager.EnumProcessModulesUntilSuccess(SafeProcessHandle processHandle, IntPtr[] modules, Int32 size, Int32& needed, Int32 filterFlag)
   at System.Diagnostics.NtProcessManager.GetModules(Int32 processId, Boolean firstModuleOnly)
   at System.Diagnostics.NtProcessManager.GetFirstModule(Int32 processId)
   at CharacterAI.Integration.KillChromes(String execPath)
   at CharacterAI.Integration.LaunchChromeAsync(String customChromeDir, String customExecPath)
   at CharacterAI_Discord_Bot.Program.LaunchChromeAndSetup(Boolean setup)_

(using chromium and its on character ai page)

**Comment by drizzle-mizzle** [comment]
It tried to clear possible running background chrome processes (embedded, not your), but you didn't give the program enough permissions to do that in your system. My bad I didn't make it safer. Never thought it will require any additional permissions inside its own folder.

If it doesn't affect the work of the bot, just ignore it. If it hangs on, you will either need to run it as administrator, or wait for an update that will be aired soon.

**Comment by massob** [comment]
I tried running it as an admin like you said, but I still got the same error, When I ignored it and tried to set a character it gave :warning: Failed to set a character! and **Fetching character info...** _You need to launch a browser first!_ 

I thought maybe disabling background apps was causing the problem so I turned them back on but it didn't help. I guess I'll just have to wait for an update. Thanks for your help!
